src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -469,25 +469,37 @@ class LazyMobXTableStore<T> {
         }
     }
 
+    @computed get clickableClassName() {
```

Comment:
seems like it would be better to leave specific color out of this and just have semantic states which could be targeted in css?